docs/usage.md

```diff
@@ -14,6 +14,45 @@ The following sections describe how to use Peerd in your Kubernetes cluster.
 - `helm` installed and configured.
 - `kubectl` installed and configured.
 
+### Overlaybd Peer-to-Peer Configuration for Artifact Streaming
+
+This step is `OPTIONAL` and should be performed only if you are using artifact streaming. Please skip directly to the next
+section if you are pulling images instead.
+
+Artifact streaming leverages the `overlaybd-snapshotter` which is well integrated with `containerd`. More information on
+`overlaybd-snapshotter` can be found [here](overlaybd-snapshotter).
```

Comment:
Is `overlaybd-snapshotter` already present in all `containerd` instances? If yes or no, we should briefly mention this before linking to other docs.